lib/tags.js

```diff
@@ -73,6 +73,7 @@ exports.parseTagLine = (line) => {
  * @public
  *
  * @param {String[]} footerTagLines - footer tag lines
+ * @param {Boolean} lowerCaseFooterTags - lowercase footer tag keys
```

Comment:
Can we accept an options object that contains this key instead, since the options passed to this function is likely to grow more in the future?